spring-web-modules/spring-rest-http-3/src/main/java/com/baeldung/bulkandbatchapi/response/CustomerBulkResponse.java

```diff
@@ -0,0 +1,46 @@
+package com.baeldung.bulkandbatchapi.response;
+
+import com.baeldung.bulkandbatchapi.request.Customer;
+import com.baeldung.bulkandbatchapi.request.BulkActionType;
+
+import java.util.List;
+
+public class CustomerBulkResponse {
+
+    private BulkActionType bulkActionType;
+    private List<Customer> customers;
+    private BulkStatus status;
+
+    public BulkActionType getBulkType() {
+        return bulkActionType;
+    }
+
+    public void setBulkType(BulkActionType bulkActionType) {
+        this.bulkActionType = bulkActionType;
+    }
+
+    public List<Customer> getCustomers() {
+        return customers;
+    }
+
+    public void setCustomers(List<Customer> customers) {
+        this.customers = customers;
+    }
+
+    public BulkStatus getStatus() {
+        return status;
+    }
+
+    public void setStatus(BulkStatus status) {
+        this.status = status;
+    }
+
+    public static CustomerBulkResponse getCustomerBatchResponse(List<Customer> customers, BulkActionType bulkActionType, BulkStatus batchStatus) {
```

Comment:
This isn't a `get`, it's a `create` function - a factory method. Though I don't see what it adds that a constructor wouldn't.

Reply:
True, can be used directly a constructor